frontend/src/__tests__/cypress/cypress/pages/home/homeProject.ts

```diff
@@ -0,0 +1,35 @@
+import { Contextual } from '~/__tests__/cypress/cypress/pages/components/Contextual';
+
+export class HomeProject extends Contextual<HTMLElement> {
```

Comment:
Is this really needing to be contextual? If so, then the child queries should be chained with find(), if not just remove the extends.